src/main/java/reactor/netty/resources/ConnectionProvider.java

```diff
@@ -111,8 +115,7 @@ static ConnectionProvider create(String name) {
 	 * number of {@link Connection}
 	 */
 	static ConnectionProvider create(String name, int maxConnections) {
-		return builder(name).maxConnections(maxConnections)
-		                    .build();
+		return builder(name).maxConnections(maxConnections).lifo();
```

Comment:
Why did you change the strategy to LIFO, we still want it FIFO by default?

Reply:
@violetagg if lifo is the default, then we won’t need an explicit switch to lifo.
we can just use .build(), or fifo() if need.

Reply:
@jdaru We say that FIFO is by default, so I would expect that the methods that do not provide Builder (the two create methods) will always create FIFO, if somebody wants LIFO then he/she needs to use the Builder and not these two methods.